### ISSUE TITLE
All pull requests have 'across 1 directory' in the title since today

### ISSUE THREAD

**Issue by mvz** [dependabot/dependabot-core#9631]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

bundler

### dependabot.yml content

For example, https://github.com/Docbldr/docbldr-use-demo/blob/2502f361346f5b7394fb8cb1804e58c32d422439/.github/dependabot.yml

### Updated dependency

selenium-webdriver from 4.19.0 to 4.20.1, and many other cases in all of our repositories

### What you expected to see, versus what you actually saw

I expected to see a pull request title 'Bump selenium-webdriver from 4.19.0 to 4.20.1 in the development-dependencies group'.

Instead the pull request title is 'Bump selenium-webdriver from 4.19.0 to 4.20.1 in the development-dependencies group across 1 directory'. (See https://github.com/Docbldr/docbldr-use-demo/pull/3029).

Since there is only one directory with dependency information, this extra text is superfluous.

The same issue happens in all of the pull requests created by dependabot since today.

**Comment by abdulapopoola** [comment]
Hi @mvz , that is because we shipped [multi-dir support](https://github.blog/changelog/2024-04-29-dependabot-multi-directory-configuration-public-beta-now-available/). We can look at improving this, tagging @carlincherry

**Comment by corneliusroemer** [comment]
I opened a [duplicate](https://github.com/dependabot/dependabot-core/issues/9639) before discovering this issue. While multi-dir support might be nice, it's important to special case the single-dir case for PR titles as this will likely remain the most common use case.

In a normal repo with multiple groups configured, the current change makes things much less user friendly.

**Comment by corneliusroemer** [comment]
@mvz you are right! In your case it's just extra noise, in my case it's loss of crucial information. I might reopen.

**Comment by mvz** [comment]
I should add that this happens intermittently. Dependabot just now even closed PRs without 'across 1 directory' and opened new ones including the offending phrase.

**Comment by abdulapopoola** [comment]
@mvz , please does this still occur?

**Comment by corneliusroemer** [comment]
Yes it happened again two days ago for 2 PRs, that only covered one directory each

**Comment by mvz** [comment]
I on the other hand have not seen it happen in the last two weeks 🤷

**Comment by mvz** [comment]
Unfortunately, this problem happened again today.

**Comment by jpveooys** [comment]
> The original pull request is fine, but during the rebase somehow dependabot decides to throw that one away and creates a new one with the incorrect message.

I have also just seen this behaviour.

**Comment by jpveooys** [comment]
Note that when Dependabot recreates the PR with 'across 1 directory' in the title, it also stops obeying the documented rule that, when grouped updates are enabled, updates go in the first matched grouped. This matters if you have a catch-all ‘other’ group – Dependabot incorrectly closes PRs and creates new PRs with 'across 1 directory' in the title with incorrect grouping rules applied. Therefore the bug is more severe than it sounds at first glance.

Unfortunately, the repo I'm seeing this in isn't a public one, but happy to share examples in private.
